libs/tup-components/src/projects/details/ProjectDetails.tsx

```diff
@@ -157,7 +182,18 @@ const ProjectDetails: React.FC<{ projectId: number }> = ({ projectId }) => {
 
   return (
     <div className={styles['pub-details-container']}>
-      <SectionTableWrapper header="Allocations">
+      <SectionTableWrapper
+        header="Allocations"
+        headerActions={
+          canManage && (
+            <span>
+              <NewAllocation />
+              {' | '}
+              <IncreaseAllocation />
+            </span>
```

Comment:
Is the `<span>` serving a purpose in the markup or CSS. If not, and it was added because React complained about needing a single element, you can use `<>` and `</>` instead.[^1]

[^1]: This is a [React fragment (short syntax)](https://reactjs.org/docs/fragments.html#short-syntax).

Reply:
the span keeps those two links on the right side of page. Using <> and </> changes their alignment. 
![Screen Shot 2023-02-10 at 1 54 49 PM](https://user-images.githubusercontent.com/35277477/218185511-c23aac22-ea2e-48bd-ac4a-814ed021d8d2.png)

Reply:
Thanks. Sorry I didn't have bandwidth to test myself. I normally do.